python-mini-projects/projects/convert_png_images_to_ico_format/convertUI.py

```diff
@@ -0,0 +1,44 @@
+import tkinter as tk
+from PIL import Image
+
+# Initialize Tkinter window
+root = tk.Tk()
+# Initialize variable to store image path
+img = None
+# Initialize font, background color, foreground color and width for the buttons
+font = ('helvetica', 12, 'bold')
+bg = 'blue'
+fg = 'white'
+width = 15
+
+def getPNG():
+    '''Function to get png image location and open it with pillow'''
+    global img
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Global variables are dangerous because they can be simultaneously accessed from multiple sections of a program. This frequently results in bugs. Most bugs involving global variables arise from one function reading and acting on the value of a global variable before another function has the chance to set it to an appropriate value. We recommend using a configuration module to mutate global state.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-share-global-variables-across-modules)